translations/translated/mudlet_en_US.ts

```diff
@@ -144,7 +144,7 @@ for all of the %n selected rooms:</numerusform>
         </translation>
     </message>
     <message numerus="yes">
-        <location filename="../../src/mudlet.cpp" line="4468"/>
+        <location filename="../../src/mudlet.cpp" line="4417"/>
         <source>Review %n update(s)...</source>
         <comment>Review update(s) menu item, %n is the the count of how many updates are available</comment>
```

Comment:
Ooops "`the the`" - I think I made a mistake in that comment for a translation in the runt American translation file... :blush:

Reply:
I have extinguished the place you mentioned, as well as the original place mentioned in there.

Will this then keep samethe  from happening in future versions? I sure hope so.